README.md

```diff
@@ -1,3 +1,8 @@
+## DEPRECATED
+
+### *CppUTest is available through [**Conan Center**](https://conan.io/center/cpputest)*.
+
+
```

Comment:
The build status below is now very irrelevant, and I think the rest of the README should just be stripped out.

Reply:
Yes, makes sense. I'll remove them.